modules/lib/lib-portal/src/main/resources/lib/xp/portal.js

```diff
@@ -16,7 +16,7 @@
  * @param {object} params Input parameters as JSON.
  * @param {string} params.path Path to the asset.
  * @param {string} [params.application] Other application to reference to. Defaults to current application.
- * @param {string} [params.type=server] URL type. Either `server` (server-relative URL) or `absolute`.
+ * @param {string} [params.type=server] URL type. Either `server` (server-relative URL) or `absolute` or `websocket`.
```

Comment:
Isn't it strange for asset/attachment/image etc. to even mention we socket?

Reply:
All these functions use the same `type`. After adding the `websocket` value that available for the rest of functions.